docs/getting-started/preview.md

```diff
@@ -0,0 +1,79 @@
+# W4H-Preview
+## Introduction
+This document is for the preview of W4H toolkit. W4H Toolkit is an ecosystem that covers most of the common features needed when using wearables in a clinical study or or trial. As studies can have very different requirements, W4H toolkit is designed to be flexible and incrementally adoptable. This toolkit provides the following components:
```

Comment:
This is about the container. Just say what it is and how to get it.

# W4H Toolkit Preview

This is a preview of the Wearables for Health (W4H) Toolkit. To run the preview:
- Get Docker...
- Install...

Then explain what it contains with links to the dashboard, notebooks, database etc.